README.md

```diff
@@ -17,36 +17,37 @@ A curated list of awesome Open Source Software made in Israel, and by Israelis.
 <br/>
 
 # Table of Contents
-- [Projects by main language](#projects-by-main-language)
-  - [angular](#angular)
-  - [assembly](#assembly)
-  - [c](#c)
-  - [css](#css)
-  - [go](#go)
-  - [html](#html)
-  - [java](#java)
-  - [javascript](#javascript)
-  - [objective-c](#objective-c)
-  - [perl](#perl)
-  - [php](#php)
-  - [python](#python)
-  - [react](#react)
-  - [ruby](#ruby)
-  - [rust](#rust)
-  - [shell](#shell)
-  - [typescript](#typescript)
-  - [vue](#vue)
-  - [unknown language](#unknown-language)
-  - [Old Projects](#old-projects)
-- [Companies](#companies)
-- [Organizations](#organizations)
-- [Podcasts](#podcasts)
-- [Individuals](#individuals)
-- [Events](#events)
-- [Meetups / User Groups](#meetups--user-groups)
-- [Israeli Tech AwesomeLists](#israeli-tech-awesomelists)
-- [Contributing](#contributing)
-- [License](#license)
+- [Table of Contents](#table-of-contents)
```

Comment:
@danielrosehill can you please remove this change to the table of contents? that's not needed

Reply:
@lirantal - It's automatic indentation from VS Code indenting everything by one space forward whenever I save the file. Driving me crazy trying to stop it from happening. Let me see if I can figure it out and create a new PR (you might be able to see from the commits that I already tried once unsuccessfully!)

Reply:
the problem is that it's indenting one level deep.

feel free to close this PR and make changes only to the other section in the README to add your project and submit a new PR with it.

Reply:
Will do!